packages/web/components/complex/portfolio/get-started-with-osmosis.tsx

```diff
@@ -0,0 +1,31 @@
+import { FunctionComponent } from "react";
+
+import { Button } from "~/components/ui/button";
+import { useTranslation } from "~/hooks";
+import { useWalletSelect } from "~/hooks/use-wallet-select";
+import { useStore } from "~/stores";
+
+export const GetStartedWithOsmosis: FunctionComponent = () => {
+  const { chainStore } = useStore();
+  const { t } = useTranslation();
+
+  const { onOpenWalletSelect } = useWalletSelect();
+
+  return (
+    <div className="flex max-w-sm flex-col gap-8">
+      <p className="body1 text-osmoverse-400">{t("portfolio.connectWallet")}</p>
+      <Button
+        className="flex !h-11 w-fit items-center gap-2 !rounded-full !py-1"
+        onClick={() => {
+          onOpenWalletSelect({
+            walletOptions: [
+              { walletType: "cosmos", chainId: chainStore.osmosis.chainId },
```

Comment:
We're considering `chainStore` to be deprecated; a part of the Keplr architecture we're trying to migrate away from. In the meantime, could we use the osmosis chain ID from the `accountStore`? I believe it's just `accountStore.osmosisChainId`